src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/CounterPayload.cs

```diff
@@ -35,7 +33,6 @@ protected CounterPayload(DateTime timestamp,
 
         public string DisplayName { get; protected set; }
 
-        public string Unit { get; }
```

Comment:
To minimize breaking changes, consider public string Unit => CounterMetadata.CounterUnit

Reply:
Added